versions.yaml

```diff
@@ -56,6 +56,10 @@ module-sets:
       - go.opentelemetry.io/collector/semconv
       - go.opentelemetry.io/collector/service
       - go.opentelemetry.io/collector/filter
+  experimental:
+    version: v0.0.1
+    modules:
+      - go.opentelemetry.io/collector/pdata/pprofile
 
```

Comment:
I think we can put this in the beta module set, unless we expect very frequent breaking changes to the data model (I assume there will be some, but since the Profiling SIG has been working on this for some time, I wouldn't expect a complete redesign, right?).

I realize now that my mention of #9209 may have been confusing, but I was thinking about how to add API in other Go modules if they are 1.0 (i.e. the part where I talk about if we need to use build tags/other approaches). 

If you still think it's better to have a different module set for this, then we would need to
- add support for this at the release automation level
- add instructions for release managers to follow when releasing that
- Define when breaking changes are allowed in this module set, when will we do releases, how will the release version increase

Reply:
Following a discussion with @mx-psi, I have moved the package to beta, as beta packages are allowed breaking changes.